.github/workflows/tvos-compile-test.yml

```diff
@@ -5,6 +5,8 @@ on:
   pull_request:
     paths:
       - .github/workflows/tvos-compile-test.yml
+      - packages/expo-updates/e2e/setup/create-eas-project-tv.ts
+      - packages/expo-updates/e2e/setup/project.ts
```

Comment:
should we add these paths to the other e2e workflows?

Reply:
I think `project.ts` should already be there in the other workflows... I will check.